packages/wallet-core/src/swaps/fastbtc/FastBTCWithdrawSwapProvider.ts

```diff
@@ -102,6 +102,10 @@ class FastBTCWithdrawSwapProvider extends SwapProvider {
       return null;
     }
 
+    const { min, max } = await this.getLimits();
+    const isQuoteAmountInTheRange = amount.gte(min) && amount.lte(max);
+    if (!isQuoteAmountInTheRange) return null;
```

Comment:
can we use brackets for one line statements, please?

Reply:
Yes pleasee, devs like me do not read after closing bracket on if statements  if ( condition ) <- Thats the end line :D

Reply:
Sure, wonder if there is a prettier/lint config for this?